app/players/dwarves/dwarf.py

```diff
@@ -0,0 +1,11 @@
+from app.players.player import Player
+
+
+class Dwarf(Player):
```

Comment:
`Dwarf` class implements not all abstract methods from `Player`, thus being abstract as well. Inherit the `Dwarf` class from `ABC` too.